src/mlpack/methods/ann/dists/normal_distribution.hpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file normal_distribution.hpp
+ * @author Atharva Khandait
+ *
+ * Definition of the Normal distribution class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_DISTRIBUTIONS_NORMAL_DISTRIBUTION_HPP
+#define MLPACK_METHODS_ANN_DISTRIBUTIONS_NORMAL_DISTRIBUTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Multiple univariate Normal(Gaussian) distributions.
```

Comment:
It would probably be a good idea to mention the `GaussianDistribution` class for the case where the covariance matrix is not diagonal.  (And it's probably also good to modify the `GaussianDistribution` comments too to mention this class.)

Reply:
I agree, users of both class should know the existence of the other class. :)

Reply:
I will add them in the commit after the next one.